### PR TITLE
v0.3.0 - Setting up Danger.js for repo

### PR DESCRIPTION
**Pull request by ashleynolan**
### Added
- Dangerfile setup on travis builds

### Changed
- Updated gulp-build-fozzie version

### CODE REVIEW

File: dangerfile.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/* global danger, fail, message */
```
Comment (DamianMullins) [inline dangerfile.js]: You can remove the eslint ignore lines below and add a single `/* eslint-disable no-console */` comment at the top instead 🛩

Comment (ashleynolan) [reply dangerfile.js]: Done

Comment (DamianMullins) [reply dangerfile.js]: The comments are still in the code below?

Comment (ashleynolan) [reply dangerfile.js]: 🤦‍♂️ Should be there now

**Pull request merged**